src/main/java/duke/ActionType.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+public enum ActionType {
```

Comment:
I think there are other actions as well. Is it better to involve LIST EXIT FIND into the ActionType?